flaml/automl.py

```diff
@@ -2028,7 +2028,7 @@ def custom_metric(
                 if gpu_per_trial == 0
                 else {"cpu": 1, "gpu": gpu_per_trial}
```

Comment:
embrace line 2027-2029 with ()

Reply:
I don't understand what you mean, why don't you make a suggestion?

Reply:
because it involves a line out of the selectable area. I meant add a ( before 2027 and a ) after 2029